index.html

```diff
@@ -2497,10 +2505,12 @@
         The set of presentations known to the user agent should be cleared when
         the user requests to "clear browsing data."
       </p>
-      <div class="issue">
-        The spec should clarify what is to happen to the set of known
-        presentations in "incognito" (private browsing context) mode.
-      </div>
+      <p>
+        When in private browsing mode ("incognito"), the set of presentations
+        known to the user agent must be initially empty, and all state
```

Comment:
I think this needs to be more specific - are we making a requirement of the controlling or receiving user agent?

What if I start a presentation from a non-incognito context - would I be allowed to connect to it from an incognito context (similar to connecting to it from another device)?